docs/mkdocs/docs/error_messages.md

```diff
@@ -65,6 +65,8 @@ For legacy reasons, the terms `symbol`, `stream`, and `stream ID` are used inter
 | Error Code | Cause                                                                  | Resolution                                                                                                                                          |
 |------------|------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
 | 7000       | The input provided by the user is invalid in some fashion.             | The resolution will depend on the nature of the incorrect input, and should be explained in the associated error message. |
+| 7001       | The input was expected to be a valid decimal string but it is not a valid decimal string.             | Pass a valid decimal string. |
+| 7002       | An unsupported character was found in a symbol name.             | We support only the ASCII characters between 32-127 inclussive. Change your symbols name so it contains only valid characters. **If you want to bypass this check, you can define an environment variable called - ARCTICDB_VersionStore_NoStrictSymbolCheck_int=1**. |
```

Comment:
Worth a test that setting the env var also sets the config int correctly?

Reply:
yep, tested it, it works